src/properties/triangles.c

```diff
@@ -93,6 +93,9 @@ int igraph_transitivity_avglocal_undirected(const igraph_t *graph,
     igraph_vector_t degree;
     igraph_vector_t triangles;
 
+    if (igraph_vcount(graph) == 0) {
```

Comment:
I was wondering why this is necessary. Can't we return a sort-of-sensible answer for null graphs? @szhorvat likes these corner cases, I'd be interested in what he has to say here :) For me, returning zero or NaN would both make sense, depending on the value of the `mode` parameter, no need to throw an error.